app/javascript/mastodon/components/media_gallery.js

```diff
@@ -24,6 +25,7 @@ class Item extends React.PureComponent {
     displayWidth: PropTypes.number,
     visible: PropTypes.bool.isRequired,
     autoplay: PropTypes.bool,
+    noDescriptionTitle: PropTypes.string,
```

Comment:
Not entirely happy with passing the description down as a prop since it will be the same in all cases. I guess it spares a few `intl` calls though.